ibm_catalog.json

```diff
@@ -850,7 +850,7 @@
             },
             {
               "name": "deploy-arch-ibm-account-infra-base",
-              "description": "Cloud automation for Account Configuration organizes your IBM Cloud account with a ready-made set of resource groups by default and, when you enable the \"with Account Settings\" option, it also applies baseline security and governance settings. ",
+              "description": "Cloud automation for Account Configuration creates resource groups by default and, if  \"with Account Settings\" is enabled, it applies baseline security and governance settings that impact cluster placement and access controls. ",
```

Comment:
This is still a big problem - your not addressing what I mentioned in the issue. This should not mention "Cloud automation for Account Configuration". We should also get content team to review ALL of the dependency description we have here. They need to be very straight forward 1 line sentences.